src/groups/semidirect_product_group.jl

```diff
@@ -272,6 +272,130 @@ function _compose!(
     return k
 end
 
+_doc_LSDP_diff_left_compose = """
+    diff_left_compose(
+        SDPG::LieGroup{𝔽,LeftSemidirectProductGroupOperation,<:ProductManifold}, X, g, h
+    ) where {𝔽}
+    diff_left_compose!(
+        SDPG::LieGroup{𝔽,LeftSemidirectProductGroupOperation,<:ProductManifold}, X, g, h
+    ) where {𝔽}
```

Comment:
```suggestion
    diff_left_compose(
        SDPG::LieGroup{𝔽,LeftSemidirectProductGroupOperation,<:ProductManifold}, g, h, X
    ) where {𝔽}
    diff_left_compose!(
        SDPG::LieGroup{𝔽,LeftSemidirectProductGroupOperation,<:ProductManifold}, Y, g, h, X
    ) where {𝔽}
```

Reply:
Ah yes, thanks :) I wrote the code of the docs when I was still a bit confused. I hope the math is better.

Reply:
You're welcome :). The math looks OK but I didn't check it thoroughly.